build

```diff
@@ -35,8 +35,8 @@ etcd_build() {
 	if [ -n "${BINDIR}" ]; then out="${BINDIR}"; fi
 	toggle_failpoints
 	# Static compilation is useful when etcd is run in a container
-	CGO_ENABLED=0 go build $GO_BUILD_FLAGS -installsuffix cgo -ldflags "$GO_LDFLAGS" -o ${out}/etcd ${REPO_PATH}/cmd/etcd || return
```

Comment:
Is there anyway that we can enabled CGO only for darwin? Maybe detect architecture in build script or make it configurable.

Linux still works fine with CGO disabled, as far as I know.